domain-models-runtime/src/main/java/org/folio/rest/persist/ddlgen/SchemaMaker.java

```diff
@@ -359,7 +359,7 @@ public static String convertDotPath2PostgresNotation(String prefix,
         }
         sb.append("'").append(pathParts[j]).append("'");
       }
-      if (index != null && stringType && !isFullText) {
+      if (index != null && stringType && !isFtIndex) {
```

Comment:
And you are sure it shouldn't be:
  && !isFulltext && !isFtIndex  
??

Reply:
That will fail the test. The purpose is to restore lower(fn_unaccent) for ginIndex. The real full text index will remain as is without those function wrappers.

Reply:
Can you add a comment why the FtIndex is created without lower(f_unaccent()) but fullText and all other require it?

Reply:
done